Bio/SeqFeature.py

```diff
@@ -2039,6 +2079,20 @@ class OneOfPosition(int, AbstractPosition):
     >>> p2 == 1901
     True
 
+    To allow pickling and unpickling of class instances, the arguments
```

Comment:
Same comment, this isn't of general interest but ``__getnewargs__`` lacks any docstring at all - this would work better there I think.